tests.py

```diff
@@ -564,35 +588,6 @@ def test_noise_sfts(self):
 
         self._check_maximum_power_consistency(writer)
 
-    def test_cosi_scaling(self):
```

Comment:
Why is this one removed?

Reply:
This is a vestige of when I considered including `cosi` into the whole mixture
in order to specify `h0eff` rather than `h0` for the line amplitude. 
But then we somehow concluded this was overly complicated, so we just removed the whole
functionality altogether.

Reply:
Makes sense.